src/plugins/discover/public/embeddable/saved_search_embeddable_component.tsx

```diff
@@ -23,17 +25,25 @@ const DiscoverDocTableEmbeddableMemoized = React.memo(DiscoverDocTableEmbeddable
 const DiscoverGridEmbeddableMemoized = React.memo(DiscoverGridEmbeddable);
 
 export function SavedSearchEmbeddableComponent({
+  savedSearch,
   searchProps,
   useLegacyTable,
   refs,
 }: SavedSearchEmbeddableComponentProps) {
   if (useLegacyTable) {
-    const docTableProps = {
-      ...searchProps,
-      refs,
-    };
-    return <DiscoverDocTableEmbeddableMemoized {...(docTableProps as DocTableEmbeddableProps)} />;
+    return (
+      <DiscoverDocTableEmbeddableMemoized
+        {...({
+          ...searchProps,
+          refs,
```

Comment:
I think `refs` is not needed anymore, if this would be the case , we could simplify the code here